test/operations.py

```diff
@@ -0,0 +1,94 @@
+from http import HTTPStatus
+import requests
+import json
+import os
+
+
+class Operations:
+    @staticmethod
+    def create_index(endpoint, index_name, auth=None, data=None):
+        response = requests.put(f'{endpoint}/{index_name}', auth=auth)
+        if response.status_code != HTTPStatus.OK:
+            print('Failed to create index')
+            print(response.text)
+        else:
+            print('Created index successfully')
+            print(response.text)
```

Comment:
I don't see what the value of these print statements (or log statements, if you did a replacement), since we're running this in a unit test framework that will print out a report of what went wrong with the test.  I can see some value in maybe emitting the `response.text` to STDOUT so that it's available if a test fails.